### DISCUSSION TITLE
Commands

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#37237] [Other Feature Feedback, Questions, & Ideas]
Is it necessary to commit the changes in the terminal?
Can't we just directly remove the files?

**Comment by godnondsilva** [comment]
While pulling changes you mean?

You could do that yeah. I would say it's more of a warning so that people know they should save their changes.

**Comment by airtower-luna** [comment]
If you want *any* changes (including file deletion) to become part of your repository, you must commit them. Whether you make the commit via the terminal (`git add` or `git rm`, `git commit`) or some GUI application doesn't matter.
